### PR TITLE
adds reference token to scoped token creation if requested

### PR DESCRIPTION
**Pull request by bb-Ricardo**
This will add the reference token the resource output fields if requested.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 7.6.1 (XXX XX, 2023). Tested on Artifactory 7.55.10
```
Comment (alexhung) [inline CHANGELOG.md:1]: Should be bumped to 7.7.0 due to new attribute

**Review by alexhung — CHANGES_REQUESTED** [review]
Version changes in CHANGELOG

**Review by alexhung — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## 7.7.0 (XXX XX, 2023). Tested on Artifactory 7.55.10
+
+IMPROVEMENTS:
+
+* resource/artifactory_scoped_token: adds reference token exposure on scoped token creation
```
Comment (danielmkn) [inline CHANGELOG.md:5]: * resource/artifactory_scoped_token: adds `reference_token` attribute for the resource.

File: pkg/artifactory/resource/security/resource_artifactory_scoped_token.go [hunk 3]
```diff
@@ -194,6 +203,11 @@ func ResourceArtifactoryScopedToken() *schema.Resource {
 			Computed:  true,
 			Sensitive: true,
 		},
+		"reference_token": {
+			Type:      schema.TypeString,
+			Computed:  true,
+			Sensitive: true,
+		},
```
Comment (danielmkn) [inline pkg/artifactory/resource/security/resource_artifactory_scoped_token.go:210]: Please add description here as well: 
```
Description: "Should a reference token also be created? Defaults to `false`",

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
+## 7.7.0 (XXX XX, 2023). Tested on Artifactory 7.55.10
```
Comment (danielmkn) [inline CHANGELOG.md:1]:
```
## 7.7.0 (April 26, 2023). Tested on Artifactory 7.55.10

**Review by danielmkn — CHANGES_REQUESTED** [review]
A couple of minor changes are needed.

File: pkg/artifactory/resource/security/resource_artifactory_scoped_token.go [hunk 5]
```diff
@@ -164,6 +166,13 @@ func ResourceArtifactoryScopedToken() *schema.Resource {
 			ForceNew:    true,
 			Description: "The token is not refreshable by default.",
 		},
+		"include_reference_token": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			ForceNew:    true,
+			Description: "Also create a reference token which can be used like an API key.",
```
Comment (danielmkn) [inline pkg/artifactory/resource/security/resource_artifactory_scoped_token.go:174]: Please add:
```
Default is `false`.

File: pkg/artifactory/resource/security/resource_artifactory_scoped_token.go (hunk 3 above)
Comment (danielmkn) [inline pkg/artifactory/resource/security/resource_artifactory_scoped_token.go:210]: PLease add 
```
Description: "Reference Token (alias to Access Token)",

**Review by danielmkn — CHANGES_REQUESTED** [review]

**Review by danielmkn — APPROVED** [review]

**Pull request merged**